modules/cluster-logging-updating-logging.adoc

```diff
@@ -120,6 +120,7 @@ $ oc get cronjob
 [source,terminal]
 ----
 NAME                     SCHEDULE       SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+curator                  */5 * * * *    False     0        <none>          56s
```

Comment:
A separate PR will remove ALL curator-related content from OpenShift 5.1 onward.

Reply:
can you explain why this is?